source/php/Component/Box/box.json

```diff
@@ -22,6 +22,17 @@
         "image": "Image object (see image component), svg or raster image.",
         "icon": "Icon name as a string."
     },
+    "types":{
+        "heading": "string",
+        "content": "string",
+        "meta": "string|array",
+        "secondaryMeta": "string|array",
+        "link": "string",
+        "ratio": "string",
+        "date": "boolean",
+        "image": "array|object|boolean",
```

Comment:
I do not think you need to be able to pass a boolean value here. If you do not want an image you can choose to pass nothing